zerver/lib/actions.py

```diff
@@ -675,6 +676,14 @@ def do_set_realm_property(realm: Realm, name: str, value: Any) -> None:
     )
     send_event(realm, event, active_user_ids(realm.id))
 
+    event_time = timezone_now()
+    RealmAuditLog.objects.create(
+        realm=realm, event_type=RealmAuditLog.REALM_PROPERTY_SET, event_time=event_time,
```

Comment:
Should we add event_type for each property of Realm?

Reply:
@mateuszmandera what are your thoughts on this?  I think it's probably simplest to just have a single `event_type`; if we feel the need to have an index on the property, we can always make that happen by adding a new column extracted from `extra_data`; I suppose the question is whether we want to do that now.  

Overall realm property changes will be rare so I don't think it much matters; scanning all the Realm property changes for a realm should be super fast as it's unlikely to ever be more than 100 events per realm.

Reply:
I'm very late to reply, but agreed, having so many new event types seems like a mess and doesn't seems like that much of an added benefit